stellar-dotnet-sdk/Server.cs

```diff
@@ -93,7 +92,7 @@ public RootResponse Root()
         /// <returns></returns>
         public Task<SubmitTransactionResponse> SubmitTransaction(Transaction transaction)
```

Comment:
We need two overloads of `SubmitTransaction` that accept a `FeeBumpTransaction`.

* `SubmitTransaction(FeeBumpTransaction tx)`
* `SubmitTransaction(FeeBumpTransaction tx, SubmitTransactionOptions options)`

These overloads should set the options for feebumptransaction to true.

Reply:
Done